blog/2024-01-12-pulsar-2023-year-in-review.md

```diff
@@ -0,0 +1,137 @@
+- Allows users to choose between different releases based on their needs for stability or new features;
+- Clarifies the release cycle for both maintainers and users;
+- Frees maintainers from spending too much time maintaining a long list of old releases.
+
+With the new release model, the Pulsar community looks to release LTS versions every 18 months, with bug fixes continuing for 24 months and security vulnerability patches supported for 36 months. See the image below for details.
+
+![](/img/pulsar-new-release-model.png)
+
+For more information, see [PIP-175](https://github.com/apache/pulsar/issues/15966) and the [Release policy](pathname:///contribute/release-policy/).
+
+## New Website
+
+The Apache Pulsar website has a fresh look! Thanks to Emidio Cardeira, Asaf Mesika, Tison Chen and Kiryl Valkovich for creating an engaging design that captures the future feel of our thriving community and next-gen solution.
+
+## Spring for Apache Pulsar
+
+[Spring for Apache Pulsar](https://spring.io/blog/2023/11/21/spring-for-apache-pulsar-1-0-0-goes-ga/) provides a `PulsarTemplate` for publishing to a Pulsar topic and a `PulsarListener` annotation for consuming from a Pulsar topic, as well as various convenience APIs for Spring developers to ramp up their development journey into Apache Pulsar. Support is also included in Spring Boot via [auto-configuration](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#messaging.pulsar).
+
+## Pulsar Admin Go Library
+
+[Pulsar Admin Go Library](https://github.com/apache/pulsar-client-go/tree/master/pulsaradmin) provides a unified Go API for managing Pulsar resources such as tenants, namespaces, topics, etc.
+
+## Quarkus Extension for Apache Pulsar
+
+[Quarkus Extension for Apache Pulsar](https://quarkus.io/guides/pulsar) provides support for Apache Pulsar through SmallRye Reactive Messaging framework. Based on Eclipse MicroProfile Reactive Messaging specification 3.0, it proposes a flexible programming model bridging CDI and event-driven.
+
+## Enhanced OTel-based metric system
+
+[PIP-264](https://github.com/apache/pulsar/blob/master/pip/pip-264.md) was completed, approved by the community, and started development. It will solve a big pain point for Pulsar users with a large number of topics - 50k up to 1M topics: observability. The Apache Pulsar community has taken a large undertaking to make OpenTelemetry Java SDK ready for very low latency systems such as Pulsar with two big features it currently develops: [Near-zero memory allocations](https://github.com/open-telemetry/opentelemetry-java/issues/5105) and [metric filtering upon collection](https://github.com/open-telemetry/opentelemetry-java/issues/6107), which was also added to [OpenTelemetry specifications](https://github.com/open-telemetry/opentelemetry-specification/issues/3324).
+
+## Oxia
```

Comment:
This is a StreamNative only development and I do not think it has been discussed in the community (meaning on dev@pulsar mailing list.)

Reply:
I'm talking about Oktia

Reply:
Oxia is NOT part of Apache Pulsar. SN has never asked for it to be part of Apache Pulsar. What SN accomplishes is not automatically part of what Apache Pulsar accomplishes.

Reply:
Hi @dave2wave, thank you for the heads-up. I completely agree with you.

I've realized that Spring for Apache Pulsar and the Quarkus Extension might face the same issue you highlighted for Oxia, given that they are not developed by Apache.

What are your thoughts on relocating these three items to a distinct 'Pulsar Ecosystem' section? We implemented a similar approach [last year](https://pulsar.apache.org/blog/2023/01/10/pulsar-2022-year-in-review/). This should help eliminate any confusion.

Reply:
Let's continue the discussion on the dev mailing list.

Reply:
I added the section to see what it would look like.